src/account.rs

```diff
@@ -5,37 +5,38 @@ use clap::Parser;
 use dialoguer::Input;
 use iota_wallet::account::AccountHandle;
 
-use crate::commands::account::{
-    balance_command, faucet_command, generate_address_command, list_addresses_command, list_transactions_command,
-    mint_native_token_command, mint_nft_command, send_command, send_micro_command, send_native_command,
-    send_nft_command, sync_account_command, AccountCli, AccountCommands,
+use crate::{
+    command::account::{
+        balance_command, consolidate_command, faucet_command, generate_address_command, list_addresses_command,
+        list_transactions_command, mint_native_token_command, mint_nft_command, send_command, send_micro_command,
+        send_native_command, send_nft_command, sync_account_command, AccountCli, AccountCommand,
+    },
+    error::Error,
 };
 
 // loop on the account prompt
-pub async fn account_prompt(account_handle: AccountHandle) {
+pub async fn account_prompt(account_handle: AccountHandle) -> Result<(), Error> {
     loop {
-        let exit = account_prompt_internal(account_handle.clone()).await;
-        if exit {
-            break;
+        if account_prompt_internal(account_handle.clone()).await? {
+            return Ok(());
         }
     }
 }
 
 // loop on the account prompt
-pub async fn account_prompt_internal(account_handle: AccountHandle) -> bool {
+pub async fn account_prompt_internal(account_handle: AccountHandle) -> Result<bool, Error> {
     let alias = {
         let account = account_handle.read().await;
         account.alias().clone()
     };
     let command: String = Input::new()
         .with_prompt(format!("Account `{}` command (h for help)", alias))
-        .interact_text()
-        .unwrap();
+        .interact_text()?;
 
     match command.as_str() {
         "h" => {
             if let Err(err) = AccountCli::try_parse_from(vec!["Account:", "help"]) {
-                let _ = err.print();
+                println!("{err}");
```

Comment:
Shouldn't this also be `log::error`?

Reply:
It looks a bit odd but this is not actually an error, it's the help usage. We could technically put it on `log::info` but it looks bad since it's multiline :/

Reply:
Ah right, then it's fine